.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,51 @@
+name: System Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch: {}
+  schedule:
+    - cron:  '00 04 * * 2-6'
+
+jobs:
+  system-tests:
+    if: ${{ github.event.pull_request.head.repo.full_name == 'DataDog/dd-trace-py' }}
```

Comment:
This is just to keep it from running (and failing) on forks?

Reply:
Any issues if it did run on a fork and someone had a `DD_API_KEY` set?

Reply:
> This is just to keep it from running (and failing) on forks?

yes, PR *from* a fork.

That's true, it'll prevent to run it directly on the fork, I'll remove it.